BrainPortal/app/controllers/nh_projects_controller.rb

```diff
@@ -32,7 +32,7 @@ def new #:nodoc:
   end
 
   def create #:nodoc:
-    attributes             = params.require_as_params(:nh_project).permit(:name, :description)
+    attributes             = params.require_as_params(:nh_project).permit(:name, :description, :public, :editor_ids => [])
```

Comment:
Addendum: I think all may be correct, since the `editor_ids=` method itself makes some check. That includes my comment in update(), below. And here, we never assign editors to a group we create freshly.

Reply:
That´s fine the check is done in the validation for editors.